### PR TITLE
moves the content of 50-default.conf to a file

### PR DESCRIPTION
**Pull request by MrMarvin**
It belongs there and will be included with the provided rsyslog.conf anyway.
For setups that do provide their own rules, the file wont be include. So it should not break backwards compatibility.

### CODE REVIEW

**Comment by iggy** [comment]
This will actually break anyone who was mostly relying on the defaults but had custom files. Not hard to fix, but you might want to throw a note in the README and/or pillar.example. Something along the lines of "if you were using minimal custom files before, you might need to add '50-default.conf' to your rsyslog:custom pillar"

**Comment by MrMarvin** [comment]
I added some comments to both README and pillar.example.

Please note that the chance of breaking someones current config is rather small as most installations that rely on including custom files should not have the need for the entries in 50-default.conf anyway. In fact it was kind of breaking the functionality here because it leads rsyslog to log both to a specified remote machine and (non alterable) to local files.

File: README.rst [hunk 1]
```diff
@@ -17,4 +18,8 @@ Available states
 ``rsyslog``
 ------------
 
-Install and configure the ``rsyslog`` package and enable the service.
+Install and configure the ``rsyslog`` package and enable the service. See the `pillar.example` file for configuration.
+
+Changelog
+================
+Recently the default rules were moved from `rsyslog.conf` to `50-default.conf`. This file will be loaded if the `rsyslog:custom` pillar isnt set. However, if your `rsyslog:custom` specifies other files to include, you must add the `50-default.conf` as well!
```
Comment (babilen) [inline README.rst]: This is missing a newline.

**Comment by MrMarvin** [comment]
No worries @iggy, your feedback is appreciated. I added a proper date.

**Comment by iggy** [comment]
Thanks!

**Pull request merged**